cli/flox/src/commands/mod.rs

```diff
@@ -727,11 +729,11 @@ pub fn detect_environment(
         // current directory or git repo, prompt for which to use.
         (Some(activated_env), Some(found)) => {
             let type_of_directory = if found.path == current_dir {
-                "current directory's flox environment"
+                "current directory"
             } else {
-                "flox environment detected in git repo"
+                "detected in git repo"
             };
-            let message = format!("Do you want to {message} the {type_of_directory} or the current active flox environment?");
+            let message = format!("{message} which environment?");
```

Comment:
I personally find the sentence struture a bit weird now.. starting with a capitalized verb and not really sounding like a full sentence.

How about

`Edit which environment?` vs `Which environment do you want to edit?`

Reply:
This feels like personal preference, it looks fine to me